src/java/org/apache/cassandra/service/paxos/uncommitted/PaxosStateTracker.java

```diff
@@ -130,8 +129,8 @@ public static PaxosStateTracker create(File[] directories) throws IOException
         boolean rebuildNeeded = !hasExistingData || forceRebuild();
 
         if (truncateBallotMetadata() && !rebuildNeeded)
-            logger.warn("{} was set, but {} was not and no rebuild is required. Ballot data will not be truncated",
-                        TRUNCATE_BALLOT_METADATA_PROP, FORCE_REBUILD_PROP);
+            logger.warn("{} was set to true, but {} was not and no rebuild is required. Ballot data will not be truncated",
```

Comment:
@Mmuzaf how do you know that it was set to true? that `if` may fail when `truncateBallotMetadata()` is false so `!rebuildNeeded` will not be even evaluated.